<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Standart</class>
 <widget class="QMainWindow" name="Standart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Standart Bank Account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/Bank.png</normaloff>:/icons/icons/Bank.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_money">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>15000</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>241</width>
      <height>206</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_bankomat"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_comunalka"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Внести кошти на рахунок</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="lineEdit_na_rahunok"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Переказ коштів</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Оплатити комуналні послуги</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Зняти готівку</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_perekaz"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_comunalka">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>Оплатити</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_bankomat">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>Зняти</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_perekaz">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>Надіслати</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_na_rahunok">
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #fff;
  border: 1px solid #dbdbdb;
  border-radius: .375em;
  color: #363636;
  font-family: &quot;Segoe UI&quot;;
  font-size: 1rem;
  height: 2.5em;
  line-height: 1.5;
  padding: calc(0.5em - 1px) 1em;
  position: relative;
  text-align: center;
  vertical-align: top;
  white-space: nowrap;
}

QPushButton::pressed {
  border-color: #4a4a4a;
  outline: 0;
}
</string>
       </property>
       <property name="text">
        <string>Внести</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/icons/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>На рахунку</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>140</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>140</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <column>
     <property name="text">
      <string>	Історія транзакцій</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/icons/icons/Invoice.png</normaloff>:/icons/icons/Invoice.png</iconset>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
